lib/manager/flux/extract.ts

```diff
@@ -1,17 +1,27 @@
 import { loadAll } from 'js-yaml';
+import { id as GithubReleasesId } from '../../datasource/github-releases';
 import { HelmDatasource } from '../../datasource/helm';
 import { logger } from '../../logger';
 import { readLocalFile } from '../../util/fs';
 import type { ExtractConfig, PackageDependency, PackageFile } from '../types';
-import type {
-  FluxManifest,
-  FluxResource,
-  HelmRelease,
-  HelmRepository,
-} from './types';
+import type { FluxManifest, FluxResource, ResourceFluxManifest } from './types';
+import { isSystemManifest } from '.';
 
-function readManifest(content: string): FluxManifest | null {
-  const manifest: FluxManifest = { releases: [], repositories: [] };
+function readManifest(content: string, file: string): FluxManifest | null {
+  if (isSystemManifest(file)) {
+    return {
+      kind: 'system',
+      file: file,
+      version: content.match(/#\s*Flux\s+Version:\s*(\S+)/)[1],
```

Comment:
You test whether it's a system manifest based on file name alone, so this could potentially not match if the file contents are not as expected?

Reply:
True, I'll add a test for this.